### PR TITLE
LINK-1929 | reset page param after deleting instance

### PR DESCRIPTION
**Pull request by jorilindell**
## Description
- Deleting the only instance from the last page of the search result causes 404 error and UI cannot display any results. Reset the page after deleting instance.

## Closes
[LINK-1929](https://helsinkisolutionoffice.atlassian.net/browse/LINK-1929)

[LINK-1929]: https://helsinkisolutionoffice.atlassian.net/browse/LINK-1929?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by terovirtanen** [comment]
**LINKEDCOMPONENTS-UI branch is deployed to platta: https://linkedcomponents-pr354-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://linkedcomponents-pr354-ui.dev.hel.ninja :laughing::tada::tada::tada:

File: src/domain/events/eventActionsDropdown/EventActionsDropdown.tsx [hunk 1]
```diff
@@ -66,6 +68,17 @@ const EventActionsDropdown: React.FC<EventActionsDropdownProps> = ({
     navigate(eventUrlWithReturnPath);
   };
 
+  const goToEventsPage = () => {
```
Comment (harriris-vincit) [inline src/domain/events/eventActionsDropdown/EventActionsDropdown.tsx:71]: The abstract features of these `goTo<x>Page` functions seem to repeat between the different components / pages. Maybe there could be just a single shared util function that takes a callback function for initializing `queryString` and navigating to a given path?

**Review by harriris-vincit — APPROVED** [review]
Looks and works good. Left one improvement suggestion, but otherwise can approve.

**Comment by terovirtanen** [comment]
**LINKEDCOMPONENTS-UI branch is deployed to platta: https://linkedcomponents-pr354-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://linkedcomponents-pr354-ui.dev.hel.ninja :laughing::tada::tada::tada:

**Pull request merged**